Libraries/acdriver/Sources/Options.cpp

```diff
@@ -32,6 +32,8 @@ parseArgument(std::vector<std::string> const &args, std::vector<std::string>::co
         return libutil::Options::Current<bool>(&_printContents, arg);
     } else if (arg == "--compile") {
         return libutil::Options::Next<std::string>(&_compile, args, it);
+    } else if (arg == "--compile-output-filename") {
```

Comment:
Can we call this just `--output` or is there a reason why the option has to be that long? Does this tool generate multiple output files?